src/components/Comments.jsx

```diff
@@ -16,35 +20,82 @@ export default function Comments() {
       .catch((err) => {
         console.log(err);
       });
-  }, []);
+  }, [articleId]);
+
+  const handleCommentSubmit = (e) => {
+    e.preventDefault();
+    if (newComment.trim() === "") {
+      setSubmitErr("Comment is empty...");
+      return;
+    }
+    setIsSubmit(true);
+
+    const comment = {
+      username: "tickle122",
+      body: newComment,
+    };
+    baseApi
+      .post(`/articles/${articleId}/comments`, comment)
+      .then((res) => {
+        setNewComment("");
+        setSubmitErr(null);
+        setComments((showComments) => [res.data.comment, ...showComments]);
+        setIsSubmit(false);
```

Comment:
If you get time it would be good to extract this call to the API into it's own function that can be imported into this file